### DISCUSSION TITLE
Automatische  Fermentationskontrolle startet nicht

### DISCUSSION THREAD

**Discussion by Spityman** [BernhardSchlegel/BierBot-Bricks#37] [Q&A]
Hallo zusammen,
ich habe ein Problem. Und zwar habe ich einen Brick mit Temperatursensor am Fermzilla sowie eine iSpindel im Fermzilla. Die Sensoren und Aktoren wurden dem Fermzilla zugewiesen (s. Bild). So weit so gut. Wenn ich jetzt die automatische Temperaturkontrolle einrichten will, kann ich den Fermzilla als Fermenter nicht auswählen, er wird mir nicht angezeigt (s. Bild). Wenn ich in den Manuellen Modus wechsel, wird mir ein Messwert angezeigt, aber der Gärverlauf verbleibt bei diesem Messwert. Was mache ich falsch?

<img width="786" alt="Bildschirmfoto 2024-11-11 um 09 34 40" src="https://github.com/user-attachments/assets/fbf7dcef-1a92-48b0-b924-2ea5b777d582">
<img width="900" alt="Bildschirmfoto 2024-11-11 um 09 33 05" src="https://github.com/user-attachments/assets/6fd625f6-9e1c-4ca4-b67e-ef2994169fed">

**Comment by BernhardSchlegel** [comment]
Hi, sieht so aus, als wäre der Fermenter bereits in Verwendung? Läuft ein manueller Vorgang? Falls nicht, bitte Bug Report per Blauem Nachrichtenknopf rechts unten schicken.

Viele Grüße,
Bernhard.
